docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -78,10 +78,13 @@ spec:
                   metallb.universe.tf/loadBalancerIPs: 172.17.0.80
               spec:
                 type: LoadBalancer
-        networkAttachments:
+        networkAttachments: <2>
         - storage
 '
 ----
++
+<1> Must match the {rhos_acro} deployment storage class.
+<2> Must match the network attachment for the previous {object_storage} configuration from the {OpenStackShort} deployment.
 
```

Comment:
Hackfest comments for reference: 

- https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRtZ2dcA
- https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRtZ2db8